don't create directories and install language bindings in non-owned directories
author: Kenneth Hoste (HPC-UGent)
update version 2.42.2: Alex Domingo (VUB)
update version 2.50.0: Christoph Siegert (Leipzig University)

diff --git a/tclpkg/Makefile.am b/tclpkg/Makefile.am
index f1a1e3d..c4ce18e 100644
--- a/tclpkg/Makefile.am
+++ b/tclpkg/Makefile.am
@@ -27,94 +27,6 @@ pkgIndex.tcl: @TCL_PKGINDEX@ @TCL_PKGINDEX_GD@ @TCL_PKGINDEX_SWIG@
 	cat @TCL_PKGINDEX@ @TCL_PKGINDEX_GD@ @TCL_PKGINDEX_SWIG@ end >$@
 	rm -f end
 
-if ENABLE_SHARED
-# Installation of softlinks into non-owned directories is done as a
-# separate step, with soft failures, so that a developer can do:
-#    ./configure --prefix=$HOME/graphviz; make; make install
-# without root privileges.
-install-data-hook:
-if WITH_LUA
-	-mkdir -p $(DESTDIR)@LUA_INSTALL_DIR@;
-	if test -w $(DESTDIR)@LUA_INSTALL_DIR@; then \
-		(cd $(DESTDIR)@LUA_INSTALL_DIR@; \
-			cp -f $(DESTDIR)$(pkgluadir)/libgv_lua.so gv.so;) \
-	else \
-		echo "Warning: @LUA_INSTALL_DIR@ is not writable."; \
-		echo "Skipping system installation of lua binding."; \
-	fi
-endif
-if WITH_PERL
-	-mkdir -p $(DESTDIR)@PERL_INSTALL_DIR@;
-	if test -w $(DESTDIR)@PERL_INSTALL_DIR@; then \
-		(cd $(DESTDIR)@PERL_INSTALL_DIR@; \
-			cp -f $(DESTDIR)$(pkgperldir)/libgv_perl.so gv.so; \
-			cp -f $(DESTDIR)$(pkgperldir)/gv.pm gv.pm;) \
-	else \
-		echo "Warning: @PERL_INSTALL_DIR@ is not writable."; \
-		echo "Skipping system installation of perl binding."; \
-	fi
-endif
-if WITH_PHP
-	-mkdir -p $(DESTDIR)@PHP_INSTALL_DIR@;
-	if test -w $(DESTDIR)@PHP_INSTALL_DIR@; then \
-		(cd $(DESTDIR)@PHP_INSTALL_DIR@; \
-			cp -f $(DESTDIR)$(pkgphpdir)/libgv_php.so gv.so;) \
-	else \
-		echo "Warning: @PHP_INSTALL_DIR@ is not writable."; \
-		echo "Skipping system installation of php binding."; \
-	fi
-	-mkdir -p $(DESTDIR)@PHP_INSTALL_DATADIR@;
-	if test -w $(DESTDIR)@PHP_INSTALL_DATADIR@; then \
-		(cd $(DESTDIR)@PHP_INSTALL_DATADIR@; \
-			cp -f $(DESTDIR)$(pkgphpdir)/gv.php gv.php;) \
-	else \
-		echo "Warning: @PHP_INSTALL_DATADIR@ is not writable."; \
-		echo "Skipping system installation of php binding."; \
-	fi
-endif
-if WITH_PYTHON
-	-mkdir -p $(DESTDIR)@PYTHON_INSTALL_DIR@;
-	if test -w $(DESTDIR)@PYTHON_INSTALL_DIR@; then \
-		(cd $(DESTDIR)@PYTHON_INSTALL_DIR@; \
-			cp -f $(DESTDIR)$(pkgpythondir)/libgv_python.so _gv.so; \
-			cp -f $(DESTDIR)$(pkgpythondir)/gv.py gv.py;) \
-	else \
-		echo "Warning: @PYTHON_INSTALL_DIR@ is not writable."; \
-		echo "Skipping system installation of python binding."; \
-	fi
-endif
-if WITH_PYTHON3
-	-mkdir -p $(DESTDIR)@PYTHON3_INSTALL_DIR@;
-	if test -w $(DESTDIR)@PYTHON3_INSTALL_DIR@; then \
-		(cd $(DESTDIR)@PYTHON3_INSTALL_DIR@; \
-			cp -f $(DESTDIR)$(pkgpython3dir)/libgv_python3.so _gv.so; \
-			cp -f $(DESTDIR)$(pkgpython3dir)/gv.py gv.py;) \
-	else \
-		echo "Warning: @PYTHON3_INSTALL_DIR@ is not writable."; \
-		echo "Skipping system installation of python3 binding."; \
-	fi
-endif
-if WITH_RUBY
-	-mkdir -p $(DESTDIR)@RUBY_INSTALL_DIR@;
-	if test -w $(DESTDIR)@RUBY_INSTALL_DIR@; then \
-		(cd $(DESTDIR)@RUBY_INSTALL_DIR@; \
-			cp -f $(DESTDIR)$(pkgrubydir)/libgv_ruby.so gv.so;) \
-	else \
-		echo "Warning: @RUBY_INSTALL_DIR@ is not writable."; \
-		echo "Skipping system installation of ruby binding."; \
-	fi
-endif
-if WITH_TCL
-	-mkdir -p $(DESTDIR)@TCL_INSTALL_DIR@;
-	if test -w $(DESTDIR)@TCL_INSTALL_DIR@/; then \
-		(cd $(DESTDIR)@TCL_INSTALL_DIR@; \
-			cp -rf $(DESTDIR)$(pkgtcldir) @PACKAGE_NAME@;) \
-	else \
-		echo "Warning: @TCL_INSTALL_DIR@ is not writable."; \
-		echo "Skipping system installation of tcl bindings."; \
-	fi
-endif
-endif
 
 # removal of installs into @xxx_INSTALL_DIR@ fail if root
 #     has installed a system copy
